Sources/AsyncAlgorithms/Channels/ChannelStateMachine.swift

```diff
@@ -10,8 +10,11 @@
 //===----------------------------------------------------------------------===//
 @_implementationOnly import OrderedCollections
 
-struct ChannelStateMachine<Element, Failure: Error>: Sendable {
-  private struct SuspendedProducer: Hashable {
+// NOTE: this is only marked as unchecked since the swift-collections tag is before auditing for Sendable
+extension OrderedSet: @unchecked Sendable where Element: Sendable { }
```

Comment:
@lorentey This is the use in question

Reply:
This is alright to do -- the type is already conditionally sendable, it just isn't marked as such in a shipping release yet.

My only worry is what happens when swift-collection ships the official conformance. Is the duplicate conformance going to become a compiler error?

Reply:
we are limiting usage to 1.0.4 so as soon as we upgrade we can remove this